concourse/pipelines/create-bosh-concourse.yml

```diff
@@ -343,7 +343,7 @@ jobs:
             type: docker-image
             source:
               repository: governmentpaas/self-update-pipelines
-              tag: 0eff5b6a9c092f865a2b19cc4e75a3b539b82fa2
+              tag: 4012d25ceb903b46908a830b8e05773ced1c8f86
```

Comment:
I don't think you meant to update this container - self-update-pipelines won't have changed, and this will only cause an unnecessary download in Concourse.